libyul/optimiser/Semantics.cpp

```diff
@@ -106,8 +106,7 @@ map<YulString, SideEffects> SideEffectsPropagator::sideEffects(
 	for (auto const& function: _directCallGraph.functionsWithLoops + _directCallGraph.recursiveFunctions())
 	{
 		ret[function].movable = false;
```

Comment:
I reverted this change. Now, `movable` is false for loops and recursive calls.

Reply:
Isn't it still there?

Reply:
Now all of them are set to false and tests (functionSideEffects) updated.